model/project_ref.go

```diff
@@ -1921,6 +1923,15 @@ func FindProjectRefs(key string, limit int, sortDir int) ([]ProjectRef, error) {
 	return projectRefs, err
 }
 
+// CanEnableRepotracker checks if the project can enable the repotracker
+// with the current state of their project settings.
+func (p *ProjectRef) ValidateModifiedRepotracker() error {
```

Comment:
The user can't modify the repotracker, that's a job that tracks the repo. We also don't know when calling this function that it's been modified. Could we tweak the title?